Updating readme to avoid permission errors. (#5)
pcasaretto/dotenv: README.md

Grab the [latest binary](https://github.com/pcasaretto/dotenv/releases/latest) and put it somewhere in your $PATH

```
curl -L https://github.com/pcasaretto/dotenv/releases/download/v1.0.4/dotenv-`uname -s`-`uname -m` -o /usr/local/bin/dotenv && chmod +x /usr/local/bin/dotenv
```

> If you got a not permited error, use `sudo`

Reviewer: ```suggestion
If you receive a permission error, you might need `sudo`
```
